app/src/userpages/components/StreamPage/index.jsx

```diff
@@ -118,9 +104,10 @@ const StreamPage = (props) => {
                 />
             ) : (
                 <Edit
-                    stream={editedStream}
+                    stream={stream}
                     canShare={canShare}
                     disabled={updating}
+                    isNewStream={isNewStream}
```

Comment:
Can we move `isNewStream` into the `Edit` file, maybe? Feels like this piece belongs there.